### PR TITLE
test(e2e,playwright): prism-formatted

### PR DESCRIPTION
**Pull request by raditotev**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #51705

camperbot cant code

### CODE REVIEW

**Comment by raditotev** [comment]
@freeCodeCamp/dev-team for some reason the screenshots required for the test to work are missing on the CI machine. I have also updated `e2e/playwright.config.ts` to create screenshots if missing but this too does not work. What is it that I am missing?

**Review by huyenltnguyen — COMMENTED** [review]
There is some accessibility enhancement that we add to the `PrismFormatted` component, via `enhancePrismAccessibility`. I'm wondering if we could cover that in the test as well.

File: e2e/playwright.config.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export default defineConfig({
     /* Use custom test attribute */
     testIdAttribute: 'data-playwright-test-label'
   },
+  updateSnapshots: 'missing',
```
[the remark this answers is not in the document]
Comment (raditotev) [reply e2e/playwright.config.ts:39]: I will give another try by removing the screenshots but really confused how we can trust the visual regression when the golden files are not under version control.

**Comment by raditotev** [comment]
>There is some accessibility enhancement that we add to the PrismFormatted component, via enhancePrismAccessibility. I'm wondering if we could cover that in the test as well.

Covered: [619f2a7](https://github.com/freeCodeCamp/freeCodeCamp/pull/52165/commits/619f2a752e6e57f6535ef0b23b10bacc67bb9530)

File: e2e/playwright.config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (raditotev) [reply e2e/playwright.config.ts]: Removing the screenshots didn't fix the problem.

**Comment by raisedadead** [comment]
Hi @raditotev - Sorry about the extremely delayed response. I request assistance from @Ahmadkashif and the @freeCodeCamp/dev-team to help you get unblocked. 

Thanks for working on this.

**Comment by raditotev** [comment]
Screenshots were part of this PR and the CI was failing. I removed them to see if this will change the outcome but it didn't. I don't add them back because there is no point wasting resources running the CI pipeline before having a better idea of what is going on.

**Pull request closed without merging**